src/main/flight/mixer.c

```diff
@@ -655,6 +655,14 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
 #define CRASH_FLIP_DEADBAND 20
 #define CRASH_FLIP_STICK_MINF 0.15f
 
+static float thrustToMotorOutput(float thrust) {
+    float vbatCompMotorOutputRange = motorOutputRange;
+    if (currentControlRateProfile->vbat_comp_type != VBAT_COMP_TYPE_OFF) {
+        vbatCompMotorOutputRange *= calculateVbatCompensationFactor();
+    }
+    return motorOutputMin + vbatCompMotorOutputRange * thrust * scaleRangef(currentControlRateProfile->thrust_linearization_level, 0, 100, 1.0f, thrust);
+}
+
 static void applyFlipOverAfterCrashModeToMotors(void)
```

Comment:
Changes to the way turtle mode works aren't really needed. When speaking to you you said that it was done to account for trying to use a 5s battery on 4s motors (meant to be used with the vbat throttle compensation). We have added a mixerConfig()->crashflip_motor_percent that should be able to do that job if someone is worried about burning a motor during turtle mode.

Reply:
I removed thrust linearization on turtle mode